test/widget_tests/file_manager_test.dart

```diff
@@ -7,10 +7,10 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:material_design_icons_flutter/material_design_icons_flutter.dart';
 import 'package:sil_app_wrapper/sil_app_wrapper.dart';
```

Comment:
should this be sil_** stull?

Reply:
app_wrapper depends on graphql_client which first needs to be published before this can be renamed and the correct version added as dependency, for now it has to depend on sil_app_wrapper